src/isconnected.py

```diff
@@ -0,0 +1,14 @@
+import subprocess
+
+"""
+The method is_connected takes the output of bt_check.sh and returns a boolean value.
```

Comment:
I know this comment is referring to is_connected, that's why it's a docstring right above it. Just say "this method"